src/test/archive/sample_db/latest_version

```diff
@@ -0,0 +1 @@
+2
```

Comment:
this is the most naive way to achieve what i wanted to achieve with my highly limited dhall skills - if requested i am happy to replace this file with a JSON object stored in the same place instead that includes more information